api/python/cellxgene_census/pyproject.toml

```diff
@@ -47,6 +47,11 @@ experimental = [
     "scikit-learn==1.2.2",
     "scikit-misc==0.2.0",
     "psutil==5.9.5",
+    "datasets~=2.14",
+    "tdigest~=0.5",
+    # Not expressible in pyproject.toml:
+    #"git+https://huggingface.co/ctheodoris/Geneformer",
+    # instead, experimental/ml/geneformer_tokenizer.py catches ImportError to ask user to install that.
```

Comment:
Should add user documentation for this. Maybe in the `GeneformerTokenizer` class docstring.  I'm realizing that no where do we mention in user docs how to install `experimental` packages, so maybe we should add that and at the same time mention this GeneFormer special-case.

Reply:
@atolopko-czi Good idea, added to the docstring. Also added a blurb about the experimental modules to cellxgene_census_docsite_installation.md (although I'm reluctant to discuss Geneformer specifically there to avoid one more place to repeat the desired git ref. I hope the specific `ImportError` message one gets at runtime suffices; as @pablo-gar pointed out, if you're going to be using this then it's probably not too surprising to have to install geneformer too)

Reply:
`ImportError` works great for now